### ISSUE TITLE
Error Writing Encrypted Parquet Files on Windows

### ISSUE THREAD

**Issue by yng-me** [duckdb/duckdb-r#136]
I'm trying to execute the code below on Windows (I'm on Windows 11):

```r
# sample keys 
key1 <- "key128"
key2 <- "0123456789112345"

conn <- DBI::dbConnect(duckdb::duckdb())
query_to_encrypt <- paste0("PRAGMA add_parquet_key('", key1, "', '", key2, "');")
DBI::dbExecute(conn, query_to_encrypt)

pq_path <- "./data/sample.parquet"
query <- paste0("COPY sample_df TO '", pq_path , "' (ENCRYPTION_CONFIG { footer_key: '", key1, "'});")

DBI::dbWriteTable(conn, name = "sample_df", value = sample_df, overwrite = T)
DBI::dbExecute(conn, query)
```

Here's the error I got:

![Screenshot 2024-04-01 115933](https://github.com/duckdb/duckdb-r/assets/11029004/05f043db-b785-4344-969c-e57f190b6bb7)

NOTE: It works seamlessly on my Mac

**Comment by krlmlr** [comment]
Thanks. What version of duckdb are you using?

This looks like https://github.com/duckdb/duckdb/pull/11069, which has been fixed in duckdb 0.10.1.

**Comment by yng-me** [comment]
Thanks, @krlmlr. I just installed the latest version and it's now working.
